tests/scripts/basic-build-test.sh

```diff
@@ -201,9 +251,18 @@ FUNCS_PERCENT=$((1000*$FUNCS_TESTED/$FUNCS_TOTAL))
 FUNCS_PERCENT="$(($FUNCS_PERCENT/10)).$(($FUNCS_PERCENT-($FUNCS_PERCENT/10)*10))"
 
 echo "Lines Tested       : $LINES_TESTED of $LINES_TOTAL $LINES_PERCENT%"
+if [ $(echo "$LINES_PERCENT < $LINE_COV_THRESHOLD" | bc) = "1" ]; then
```

Comment:
So I checked the POSIX standard [here](http://pubs.opengroup.org/onlinepubs/000095399/utilities/bc.html) and for this to be portable the expression should be something like:
```sh
BC_CODE="scale = 20 ; if ($LINES_PERCENT < $LINE_COV_THRESHOLD) { 1 } ; if ($LINES_PERCENT >= $LINE_COV_THRESHOLD) { 0 } ;"
if [ $(echo "$BC_CODE" | bc) = "1" ]; then
```